product-specifications/example-3.md

```diff
@@ -0,0 +1,47 @@
+# Project Specification: 'Maze Runner' CLI Game 🏃
+
+## Overview
+
+'Maze Runner' is a text-based, Command-Line Interface (CLI) game designed for users who appreciate the simplicity and challenge of classic text-based games. The main objective is to navigate a maze and find the exit as quickly as possible.
+
+## Goals
+
+1. To provide an engaging gaming experience to users who enjoy text-based games.
+2. To challenge players with randomly generated mazes of varying complexity.
+3. To include a scoring system based on maze complexity and completion time.
+
+## Scope
+
+The game will consist of the following components:
+
+1. Maze Generation: Rules for generating mazes of different sizes and complexity.
+2. User Interface: Text-based instructions and input prompts for player actions.
+3. Scoring System: Based on maze complexity and completion time.
+
+## Requirements
+
+### Maze Generation
+
+1. The game should be able to generate square mazes of different sizes, ranging from simple (e.g., 5x5 grid) to complex (e.g., 20x20 grid).
```

Comment:
players are able to choose complexity of there levels...beginner is 5*5 and professional 20*20

Reply:
Good point, this would keep players interested as they get better!